cmd/ably-boomer/main.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ably-forks/boomer"
+	"github.com/ably/ably-boomer/perf"
+	"github.com/ably/ably-boomer/tasks/ably"
+	"github.com/inconshreveable/log15"
+	"github.com/urfave/cli/v2"
+)
+
+var log = log15.New()
+
+func nameToEnvVarNames(name, prefix string) []string {
+	envVar := strings.ToUpper(name)
+	envVar = strings.ReplaceAll(envVar, "-", "_")
+	envVar = fmt.Sprintf("%s_%s", prefix, envVar)
+	return []string{envVar}
+}
+
+func generateEnvVarNames(prefix string, flags []cli.Flag) {
+	for _, flag := range flags {
+		switch f := flag.(type) {
+		case *cli.BoolFlag:
+			f.EnvVars = nameToEnvVarNames(f.Name, prefix)
```

Comment:
Why aren't these just defined directly on the flags themselves?

Reply:
Just to prevent many copy-and-paste operations (as well as setting uppercase, adding the right prefix, replacing dashes by underscores, etc.).

Reply:
I'd rather they were just defined on the flags

Reply:
I see what you mean. Looking in the code for a particular env var would also be a bit easier I suppose. I'll make the change.

Reply:
also you've removed all the ABLY_ prefixes from the existing env vars

Reply:
Changed in 30764c6.

Reply:
> also you've removed all the ABLY_ prefixes from the existing env vars

I'm not sure I understand. You mean in the flag names, there is no ably- prefix? The env vars still have it.